test/NeoSharp.Core.Test/Network/UtServer.cs

```diff
@@ -19,40 +15,28 @@ namespace NeoSharp.Core.Test.Network
     [TestClass]
     public class UtServer : TestBase
     {
-        private EndPoint _peerEndPoint;
-
-        [TestInitialize]
-        public void Initialize()
-        {
-            var networkConfig = GetNetworkConfig("tcp://localhost:8081");
-            _peerEndPoint = networkConfig.PeerEndPoints[0];
-
-            AutoMockContainer.Register(networkConfig);
-
-            var blockchainMock = AutoMockContainer.GetMock<IBlockchain>();
-
-            blockchainMock
-                .SetupGet(x => x.CurrentBlock)
-                .Returns(new Block());
-        }
-
         [TestMethod]
         public void Start_ValidNetworkConfiguration_ConnectToPeerAsStartListeningForPeers()
         {
             // Arrange 
+            var endpoint = new EndPoint { Host = "localhost", Protocol = Protocol.Tcp, Port = 8081 };
```

Comment:
Any specific reason for:
1. Every test grew up back by repeatable 3 lines
2. Blockchain mocking was dropped
?

Reply:
See answer below